<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>dataModificationWidget</class>
 <widget class="QWidget" name="dataModificationWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>640</width>
    <height>480</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Modificar Datos</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QGroupBox" name="filterGroupBox">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Maximum">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="title">
      <string>Filtro:</string>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>Tipo:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="typeComboBox"/>
      </item>
      <item>
       <widget class="QLabel" name="label">
        <property name="text">
         <string>Fecha:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="dateComboBox"/>
      </item>
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QPushButton" name="loadButton">
        <property name="text">
         <string>Cargar</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="dataGroupBox">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="title">
      <string>Datos:</string>
     </property>
     <property name="flat">
      <bool>true</bool>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <property name="topMargin">
         <number>0</number>
        </property>
        <item>
         <spacer name="horizontalSpacer_2">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QLabel" name="label_3">
          <property name="text">
           <string>Tipo:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="kindComboBox">
          <item>
           <property name="text">
            <string>Normal</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Mensual</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Solsticio Verano</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Solsticio Invierno</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Equinoccio Primavera</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Equinoccio Oto√±o</string>
           </property>
          </item>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_8">
        <property name="rightMargin">
         <number>0</number>
        </property>
        <item>
         <layout class="QFormLayout" name="formLayout">
          <item row="0" column="0">
           <widget class="QLabel" name="tempMaxLabel">
            <property name="text">
             <string>Temp Max:</string>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QDoubleSpinBox" name="tempMaxDoubleSpinBox">
            <property name="minimum">
             <double>-99990000.000000000000000</double>
            </property>
           </widget>
          </item>
          <item row="1" column="0">
           <widget class="QLabel" name="tempMinLabel">
            <property name="text">
             <string>Temp Min:</string>
            </property>
           </widget>
          </item>
          <item row="1" column="1">
           <widget class="QDoubleSpinBox" name="tempMinDoubleSpinBox">
            <property name="minimum">
             <double>-990000.000000000000000</double>
            </property>
           </widget>
          </item>
          <item row="2" column="0">
           <widget class="QLabel" name="presionLabel">
            <property name="text">
             <string>Precipitacion:</string>
            </property>
           </widget>
          </item>
          <item row="2" column="1">
           <widget class="QDoubleSpinBox" name="precipitacionSpinBox">
            <property name="minimum">
             <double>-99.000000000000000</double>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QFormLayout" name="formLayout_2">
          <property name="fieldGrowthPolicy">
           <enum>QFormLayout::AllNonFixedFieldsGrow</enum>
          </property>
          <item row="0" column="0">
           <widget class="QLabel" name="velVientoLabel">
            <property name="text">
             <string>Vel. Viento:</string>
            </property>
           </widget>
          </item>
          <item row="1" column="0">
           <widget class="QLabel" name="dirVientoLabel">
            <property name="text">
             <string>Dir. Viento:</string>
            </property>
           </widget>
          </item>
          <item row="1" column="1">
           <widget class="QSpinBox" name="dirVientoSpinBox">
            <property name="minimum">
             <number>-99</number>
            </property>
            <property name="maximum">
             <number>32</number>
            </property>
           </widget>
          </item>
          <item row="2" column="0">
           <widget class="QLabel" name="data1000500Label">
            <property name="text">
             <string>1000/500</string>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QDoubleSpinBox" name="velVientoDoubleSpinBox">
            <property name="minimum">
             <double>-99.000000000000000</double>
            </property>
            <property name="maximum">
             <double>999.990000000000009</double>
            </property>
           </widget>
          </item>
          <item row="2" column="1">
           <widget class="QSpinBox" name="data1000500SpinBox">
            <property name="minimum">
             <number>-99</number>
            </property>
            <property name="maximum">
             <number>9999</number>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QPlainTextEdit" name="observacionTextEdit"/>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_9">
        <item>
         <widget class="QPushButton" name="delButton">
          <property name="text">
           <string>Borrar</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QDialogButtonBox" name="buttonBox">
          <property name="standardButtons">
           <set>QDialogButtonBox::Cancel|QDialogButtonBox::Save</set>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
